[2022-06-15 19:14:30,415] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: loan_pred.save_data_task scheduled__2022-06-14T21:24:00+00:00 [queued]>
[2022-06-15 19:14:30,439] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: loan_pred.save_data_task scheduled__2022-06-14T21:24:00+00:00 [queued]>
[2022-06-15 19:14:30,440] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 19:14:30,440] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-15 19:14:30,441] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 19:14:30,494] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-14 21:24:00+00:00
[2022-06-15 19:14:30,499] {standard_task_runner.py:52} INFO - Started process 658 to run task
[2022-06-15 19:14:30,504] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'loan_pred', 'save_data_task', 'scheduled__2022-06-14T21:24:00+00:00', '--job-id', '1622', '--raw', '--subdir', 'DAGS_FOLDER/loan_pred.py', '--cfg-path', '/tmp/tmpcjn1mtzx', '--error-file', '/tmp/tmpvwj5nph2']
[2022-06-15 19:14:30,506] {standard_task_runner.py:80} INFO - Job 1622: Subtask save_data_task
[2022-06-15 19:14:30,638] {task_command.py:370} INFO - Running <TaskInstance: loan_pred.save_data_task scheduled__2022-06-14T21:24:00+00:00 [running]> on host DESKTOP-GDPK8BL.localdomain
[2022-06-15 19:14:31,210] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=loan_pred
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-14T21:24:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-14T21:24:00+00:00
[2022-06-15 19:14:31,223] {python.py:173} INFO - Done. Returned value was: None
[2022-06-15 19:14:31,247] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=loan_pred, task_id=save_data_task, execution_date=20220614T212400, start_date=20220615T171430, end_date=20220615T171431
[2022-06-15 19:14:31,322] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-15 19:14:31,468] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
